tests/storage/storage.py

```diff
@@ -192,3 +192,30 @@ def coalesce_integrity(vm: VM, vdi: VDI, vdi_op: CoalesceOperation):
         vm.disconnect_vdi(vdi)
         if new_vdi is not None:
             new_vdi.destroy()
+
+XVACompression = Literal['none', 'gzip', 'zstd']
+
+def xva_export_import(vm: VM, compression: XVACompression):
```

Comment:
Why not use the fixture that already install randstream?

Reply:
I haven't read the details, but in the general case, tests that require a unix VM should depend on the `unix_vm` fixture, directly or indirectly. Which has two benefits:
1. Fail early if misused
2. `unix_vm` is one of the fixtures that we also export as markers, so job definitions in job.py can use it to select tests depending on the target platform (unix vs windows).